py_ext/py_ext.py

```diff
@@ -83,24 +82,28 @@ def CSVRdStr(fname, rowname, colname):
     if entry is None:
         data = dict()
         try:
-            csvfile = open(fname, 'rb')
+            csvfile = open(fname, 'rt', encoding='utf-8')
         except IOError:
             return "#FILE_NOT_FOUND"
         try:
             dialect = csv.Sniffer().sniff(csvfile.read(1024))
             csvfile.seek(0)
             reader = csv.reader(csvfile, dialect)
-            headers = dict([(name, index) for index, name in enumerate(reader.next()[1:])])
+            first_row = True
```

Comment:
Why is this change ? Isn't csv.reader an iterator ?

Reply:
The function `next()` no longer exists, so I initially just used the loop through rows. I've now replaced this with `__next__()` function